torch/fx/node.py

```diff
@@ -468,7 +468,9 @@ def format_node(self,
     @compatibility(is_backward_compatible=True)
     def replace_all_uses_with(self,
```

Comment:
this is an FX public API change (it shouldn't be BC breaking though) - so I'll have to update the tests.

Who else should review public FX API changes? @SherlockNoMad ?

Reply:
This is not a BC breaking change. So it should be fine.